en/apropos.md

```diff
@@ -0,0 +1,26 @@
+
+We were able to scale this product thanks to strong collaboration between different government agencies and the greater public. Indeed, we started with 7 target benefits and only 70 tests provided by partner agencies. We now compute 22 benefits and have over 600 tests, including edge cases, provided by new partner agencies, foreign territories, but also NGOs using the tool on a daily basis, and even individuals in specific situations. And once again, by opening the code powering this service, strong opportunities were enabled. Two new governmental services, independently from the Incubator, will use the same computation engine to target specific populations. During the Open Government Partnership worldwide summit hackathon, teams of 4 could make tax law computable for countries such as Tunisia and Senegal under 48 hours.
+
+Another service incubated at beta.gouv.fr uses that same engine to provide an embeddable widget that computes the cost of hiring an employee, with all taxes and refunds included, with a single line of code. The first public version of that tool was once again delivered under 6 months and one full-time person. It is now used in over 8 different partner websites, from administrations, startups and SMEs all alike.
+
+Our continued growth, our now 27 products, as well as our increasing number of partner ministries, all prove that building digital services for and with the public, ensuring collaboration across government agencies and beyond, ignoring usual committee-based, siloed decision-making, yields greatly improved services at a fraction of the cost and fosters interest and support from the civil society.
+
+## Lessons learnt
+
+We have learnt that the best strategy in digital transformation is delivery. Rather than trying to convince other stakeholders —be they fellow administrations, the private sector or individuals— to work towards a specific goal, it is more efficient to start working on it, while opening at the same time as many ways for contribution as possible. As feedback is truly listened to and taken action upon, a community starts shaping itself and the power of the crowd is leveraged. That support can then be shifted into resources, but also into political power to support larger transformation efforts, convincing more reluctant actors to consider digital transformation as an opportunity rather than a challenge.
```

Comment:
Typesetting of an em-dash looks different in a non-monospace font. I'm ok for removing the external spaces if you care, I'm used to following setting close (i.e. no space) as a typographic best practice, compensated by screenreader compatibility which makes me add external spaces.